examples/jsm/renderers/webgl/nodes/WebGLNodeBuilder.js

```diff
@@ -510,6 +528,18 @@ class WebGLNodeBuilder extends NodeBuilder {
 
 	}
 
+	getInstanceIndex() {
+
+		const node = uint( viewportCoordinate.y ).mul( viewportResolution.width ).add( uint( viewportCoordinate.x ) );
```

Comment:
Not sure why these uints are needed -- I fixed ViewportNode so that it produces uvec viewportCoordinate and viewportResolution...